### ISSUE TITLE
Cannot access system via SSH after activation

### ISSUE THREAD

**Issue by vstrelk** [thendawg/nvezos#4]
After clean install of 1.02 folled by activation, everything is working except access to system via SSH. Because of the headless config this is the only way to configure additional hw (secondary nic).
Port scanner also shows no other open ports except 443.

UPDATE: I've fixed it by manually installing ssh BEFORE the "activation".

**Comment by thendawg** [comment]
Yeah this seems to be an issue with the auto-install of ssh on 16.04.3, due to this issue being reported numerous times, Im reverting the base image back to 16.04.2 to insure this isnt an issue. (16.04.3 didnt really seem to provide any benefit anyways)
